src/__tests__/which_os_test.js

```diff
@@ -0,0 +1,69 @@
+var whichOs = require('../which_os');
+
+describe('whichOs', function () {
+  it('should returns Android as OS', function () {
+    var androidUA = [
+      'Mozilla/5.0 (Linux; U; Android 4.0.3; ko-kr; LG-L160L Build/IML74K) AppleWebkit/534.30 (KHTML, like Gecko) Version/4.0 Mobile Safari/534.30',
+      'Mozilla/5.0 (Linux; U; Android 4.0.3; de-ch; HTC Sensation Build/IML74K) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Mobile Safari/534.30',
+      'Mozilla/5.0 (Linux; U; Android 2.3; en-us) AppleWebKit/999+ (KHTML, like Gecko) Safari/999.9'
+    ].forEach(function (userAgent) {
+      expect(whichOs(userAgent)).to.equal('android');
+    })
+  });
+
+  it('should returns Blackberry as OS', function () {
+    var bbUA = [
+      'Mozilla/5.0 (BB10; <Device Model>) AppleWebKit/<WebKit Version> (KHTML, like Gecko) Version/<BB Version #> Mobile Safari/<WebKit Version>'
+    ].forEach(function (userAgent) {
+      expect(whichOs(userAgent)).to.equal('blackberry');
+    })
+  });
+
+  it('should returns iPhone as OS', function () {
+    var iphoneUA = [
+      'Mozilla/5.0 (iPhone; U; CPU like Mac OS X; en) AppleWebKit/420+ (KHTML, like Gecko) Version/3.0 Mobile/1C25 Safari/419.3',
+      'Mozilla/5.0 (iPhone; U; CPU iPhone OS 4_0 like Mac OS X; en-us) AppleWebKit/532.9 (KHTML, like Gecko) Version/4.0.5 Mobile/8A293 Safari/6531.22.7',
+      'Mozilla/5.0 (iPod touch; CPU iPhone OS 7_0_3 like Mac OS X) AppleWebKit/537.51.1 (KHTML, like Gecko) Version/7.0 Mobile/11B511 Safari/9537.53'
+    ].forEach(function (userAgent) {
+      expect(whichOs(userAgent)).to.equal('iphone');
+    })
+  });
+
+  it('should returns iMac as OS', function () {
+    var macUA = [
+      'Mozilla/5.0 (iPad; U; CPU OS 3_2 like Mac OS X; en-us) AppleWebKit/531.21.10 (KHTML, like Gecko) Version/4.0.4 Mobile/7B334b Safari/531.21.10',
+      'Mozilla/5.0 (iPad; CPU OS 7_0 like Mac OS X) AppleWebKit/537.51.1 (KHTML, like Gecko) Version/7.0 Mobile/11A465 Safari/9537.53',
+      'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_7_4) AppleWebKit/534.57.2 (KHTML, like Gecko) Version/5.1.7 Safari/534.57.2'
+    ].forEach(function (userAgent) {
+      expect(whichOs(userAgent)).to.equal('mac');
+    })
+  });
+
+  it('should returns Windows as OS', function () {
```

Comment:
What do you think about generally using the exact string which will be returned to the example description? So there's no confusion what the method will actually return